client/src/app-components/page-size-dropdown.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+import DropDown from 'core-components/drop-down';
+
+class PageSizeDropdown extends React.Component {
+    static propTypes = {
+        value: React.PropTypes.number,
+        onChange: React.PropTypes.func,
+        pages: React.PropTypes.array
+    }
+
+    state = {
+        selectedIndex: 1
+    }
+
+    static defaultProps = {
+        showDropDown: true
+    }
+
+    render() {
+        return (
+            this.props.showDropDown ?
```

Comment:
This component should know nothing about this `showDropDown`. The intuition is that you shouldn't try to render a component that is basically an empty div.

Instead, the parent component should conditionally render the entire `PageSizeDropdown` component, depending on that boolean value.